internal/common/tekton.go

```diff
@@ -0,0 +1,80 @@
+package common
+
+import (
+	"fmt"
+
+	pipeline "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	v1 "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func RemoveTektonResource(objects []client.Object, request *Request) error {
+	for _, resource := range objects {
+		err := request.Client.Delete(request.Context, resource)
+		if errors.IsNotFound(err) {
+			continue
+		}
+		if err != nil {
+			request.Logger.Error(err, fmt.Sprintf("Error deleting tekton resource \"%s\": %s", resource.GetName(), err))
+			return err
+		}
+	}
+	return nil
+}
+
+func GetPipelinesManagedByTTO(request *Request) ([]pipeline.Pipeline, error) {
```

Comment:
Will these funcs get objects in all namespaces?

Reply:
Good question. I think if there is no namespace specified, it should search in all namespaces.